src/server/services/wms/qgswmsrendercontext.cpp

```diff
@@ -287,6 +287,19 @@ bool QgsWmsRenderContext::isValidLayer( const QString &nickname ) const
   return layer( nickname ) != nullptr;
 }
 
+QList<QgsMapLayer *> QgsWmsRenderContext::layersFromGroup( const QString &nickname ) const
+{
+  QList<QgsMapLayer *> layers;
+  if ( mLayerGroups.contains( nickname ) )
+  {
+    for ( QgsMapLayer *layer : mLayerGroups[nickname] )
```

Comment:
Shouldn'it is be  qgis::as_const( mLayerGroups[nickname] ) ?

Reply:
Do I have to update the method signature ?

Do I firstly make the changes on master ?